gui/velociraptor/src/components/forms/form.jsx

```diff
@@ -449,7 +449,7 @@ export default class VeloForm extends React.Component {
                       placeholder={T("Choose one or more items")}
                       className="velo"
                       classNamePrefix="velo"
-                      closeMenuOnSelect={false}
+                      closeMenuOnSelect={true}
```

Comment:
I think this will be annoying- when we leave the menu open we can just select multiple items quickly otherwise we will need to click again to open it each time .

For example the category selector in the registry hunter

Reply:
Yes I agree. I just tried it with RegistryHunter and there it's definitely better that it stays open. I hadn't considered such long selection lists. I'll revert it.